tools/nut-scanner/nut-scanner.c

```diff
@@ -305,6 +316,15 @@ int main(int argc, char *argv[])
 					timeout = DEFAULT_TIMEOUT*1000*1000;
 				}
 				break;
+                        case 'T' :
+#ifdef HAVE_PTHREAD
+                                thread_number = atol(optarg);
```

Comment:
* Indentation is different from neighboring code.
* `atol()` does not check for errors (e.g. user passes a string or empty token), maybe `strtol()` is better here?